bigbluebutton-html5/client/stylesheet/tailwind.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
fix eslint

Reply:
it is vscode css lint warning of unknown at rules.
Please see this: https://laracasts.com/discuss/channels/code-review/vs-code-errors-and-warnings-when-using-at-tailwind-directives?page=0

Reply:
I mean make the last line empty, that requires a newline at the end of each file

Reply:
done